pkg/user/tx_client.go

```diff
@@ -415,6 +441,54 @@ func (client *TxClient) broadcastTx(ctx context.Context, txBytes []byte, signer
 	return resp.TxResponse, nil
 }
 
+// broadcastMulti broadcasts the transaction to multiple connections concurrently
+// and returns the response from the first successful broadcast.
+func (client *TxClient) broadcastMulti(ctx context.Context, txBytes []byte, signer string) (*sdktypes.TxResponse, error) {
+	respCh := make(chan *sdktypes.TxResponse, 1)
+	errCh := make(chan error, len(client.conns))
+
+	g, childCtx := errgroup.WithContext(ctx)
+
+	for _, conn := range client.conns {
+		g.Go(func() error {
+			resp, err := client.broadcastTx(childCtx, conn, txBytes, signer)
+			if err != nil {
+				errCh <- err
+				return nil
+			}
+
+			select {
+			case respCh <- resp:
+			case <-childCtx.Done():
+			}
+			return nil
+		})
+	}
+
+	groupErr := g.Wait()
+
+	close(respCh)
+	close(errCh)
+
+	if firstResp, ok := <-respCh; ok {
+		for err := range errCh {
+			fmt.Printf("Ignored broadcast error after success: %v", err)
```

Comment:
I would avoid doing anything with other errors right now - we will implement metrics later that will make this more useful than just dumping debug log.